src/doc/guide.md

```diff
@@ -746,11 +736,7 @@ fn print_sum(x: int, y: int) {
 }
 ```
 
-You separate arguments with a comma, both when you call the function, as well
-as when you declare it.
-
-Unlike `let`, you _must_ declare the types of function arguments. This does
-not work:
```

Comment:
Hm. What if we just removed the "this doesn't work" bit? I think I agree that the line works better here if we use it to introduced this bad code. But every time I see a "don't do this" or "this won't work" I think: "well then why mention it?"

Therefore, how about we nuke this "don't do this" example?